CONTRIBUTING.md

```diff
@@ -11,6 +11,15 @@ request](https://github.com/pyplati/platipy/compare) to merge your changes into
 (**master**). At this point the automated tests will run and maintainers will review your
 submission before merging.
 
+## Example Installation
```

Comment:
Thanks @SimonBiggs for adding this in!

Reply:
My pleasure :) The best way to grow documentation I think is to have people try and use it and then contribute where they found gaps :)